x-pack/plugins/actions/server/builtin_action_types/servicenow/service_itom.ts

```diff
@@ -4,41 +4,28 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
-import axios from 'axios';
 
-import { Logger } from '@kbn/core/server';
-import {
-  ExternalServiceCredentials,
-  SNProductsConfigValue,
-  ServiceFactory,
-  ExternalServiceITOM,
-  ExecutorSubActionAddEventParams,
-} from './types';
+import { ServiceFactory, ExternalServiceITOM, ExecutorSubActionAddEventParams } from './types';
 
-import { ServiceNowSecretConfigurationType } from './types';
 import { request } from '../lib/axios_utils';
-import { ActionsConfigurationUtilities } from '../../actions_config';
 import { createExternalService } from './service';
 import { createServiceError } from './utils';
 
 const getAddEventURL = (url: string) => `${url}/api/global/em/jsonv2`;
 
-export const createExternalServiceITOM: ServiceFactory<ExternalServiceITOM> = (
-  credentials: ExternalServiceCredentials,
-  logger: Logger,
-  configurationUtilities: ActionsConfigurationUtilities,
-  serviceConfig: SNProductsConfigValue
-): ExternalServiceITOM => {
-  const snService = createExternalService(
+export const createExternalServiceITOM: ServiceFactory<ExternalServiceITOM> = ({
+  credentials,
+  logger,
+  configurationUtilities,
+  serviceConfig,
+  axiosInstance,
+}): ExternalServiceITOM => {
+  const snService = createExternalService({
     credentials,
     logger,
     configurationUtilities,
-    serviceConfig
-  );
-
-  const { username, password } = credentials.secrets as ServiceNowSecretConfigurationType;
```

Comment:
I saw that an axios instance was created when calling createExternalService and then another one was created here. I refactored the axios instance creation into create_service_wrapper and updated these service factory functions to use the passed-in axios instance.